content/open-source/concepts/graphql/graphql.md

```diff
@@ -37,6 +39,52 @@ A common GraphQL workflow can be seen below:
 2. To retrieve that data, a function on the GraphQL server is invoked.
 3. Lastly, GraphQL server returns the specific data requested by the client.
 
+In a system like REST, you can only pass a single set of arguments - the query parameters and URL segments of your request. But in GraphQL, every field and nested object have its own set of arguments, making GraphQL a complete replacement for making multiple API fetches. By passing arguments into scalar fields, you can implement data transformations on the server rather than on every individual client, which can be more efficient.
+
+![source="https://graphql.org/learn/queries/"](https://raw.githubusercontent.com/Codecademy/docs/main/media/GraphQL-parameters-example.png)
+
+In GraphQL, object types are the most basic building blocks of a schema and represent the kinds of objects that can be fetched from the service, along with their fields. In the GraphQL schema language, they can be represented as follows:
+
+![source="https://graphql.org/learn/queries/"](https://raw.githubusercontent.com/Codecademy/docs/main/media/GraphQL-default-values-example.png)
+
+- The "Character" type is an example of an object type, it has a set of fields that define its characteristics.
```

Comment:
This is a structural comment: wrap any code in backticks like `Character` and `String`, instead of double quotes.